less-print/viewdetails.less

```diff
@@ -1,5 +1,90 @@
 .view-details-associations, .view-details-comments, .view-details-photos,
-.scroll-top-btn, .view-details-map, .location-static, .badge, .icons-header,
-.logo {
-  display: none;
+.scroll-top-btn, .view-details-map, .badge, .icons-header {
+  display: none !important;
+}
+
+.view-details-title{
+	display: block;
+	font-size: .75em !important;
+	margin-top: 0px;
+	margin-bottom: 10px;
+}
+
+.description , .view-details-info {
```

Comment:
Adding `display: block !important` to `.description` causes the `view-details-photos` to be shown, which is not what we want (and is part of the problem you mentionned)
But I don't understand why the `description` class is added to the div... So I think we should remove the class and your proposal should be fine.
Could you try this and see how it impacts the rendering?